src/main/java/org/embulk/output/databricks/DatabricksAPIClient.java

```diff
@@ -11,9 +12,18 @@
 
 public class DatabricksAPIClient {
   public static DatabricksAPIClient create(DatabricksOutputPlugin.DatabricksPluginTask task) {
+    SetUserAgent(task);
+
     return new DatabricksAPIClient(createDatabricksConfig(task));
   }
 
+  private static void SetUserAgent(DatabricksOutputPlugin.DatabricksPluginTask task) {
+    String name = task.getUserAgent().get().get("product_name");
+    String version = task.getUserAgent().get().get("product_version");
+
+    UserAgent.withProduct(name, version);
```

Comment:
[FYI]
[UserAgent.withProduct: defined here](https://github.com/databricks/databricks-sdk-java/blob/22e2561db98389f7d94fb353696c08709925265b/databricks-sdk-java/src/main/java/com/databricks/sdk/core/UserAgent.java#L37) and [set header in client](https://github.com/databricks/databricks-sdk-java/blob/22e2561db98389f7d94fb353696c08709925265b/databricks-sdk-java/src/main/java/com/databricks/sdk/core/ApiClient.java#L256-L257)